homeassistant/components/eheimdigital/light.py

```diff
@@ -0,0 +1,157 @@
+
+        new_classic_led_ctrl_devices = [
+            device
+            for device in coordinator.hub.devices.values()
+            if isinstance(device, EheimDigitalClassicLEDControl)
+            and device.mac_address not in known_devices
+        ]
+
+        known_classic_led_ctrl_devices = [
+            device
+            for device in coordinator.hub.devices.values()
+            if isinstance(device, EheimDigitalClassicLEDControl)
+            and device.mac_address in known_devices
+        ]
+
+        for device in new_classic_led_ctrl_devices:
+            for channel in range(2):
+                if len(device.tankconfig[channel]) > 0:
+                    entities.append(
+                        EheimDigitalClassicLEDControlLight(coordinator, device, channel)
+                    )
+                    known_devices.add(device.mac_address)
+        for device in known_classic_led_ctrl_devices:
+            for channel in range(2):
+                if len(device.tankconfig[channel]) == 0:
+                    for entity in existing_entities:
+                        if entity.unique_id == f"{device.mac_address}_{channel}":
+                            registry.async_remove(entity.entity_id)
+                elif (
+                    len(
+                        [
+                            entity
+                            for entity in existing_entities
+                            if entity.unique_id == f"{device.mac_address}_{channel}"
+                        ]
+                    )
+                    == 0
+                ):
+                    entities.append(
+                        EheimDigitalClassicLEDControlLight(coordinator, device, channel)
+                    )
+
+        async_add_entities(entities)
+
+    entry.async_on_unload(coordinator.async_add_listener(check_devices))
+    check_devices()
+
+
+class EheimDigitalClassicLEDControlLight(
+    EheimDigitalEntity[EheimDigitalClassicLEDControl], LightEntity
+):
+    """Represent a EHEIM Digital classicLEDcontrol light."""
+
+    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
+    _attr_color_mode = ColorMode.BRIGHTNESS
+    _attr_effect_list = [EFFECT_DAYCL_MODE]
+    _attr_supported_features = LightEntityFeature.EFFECT
+    _attr_translation_key = "channel"
+
+    def __init__(
+        self,
+        coordinator: EheimDigitalUpdateCoordinator,
+        device: EheimDigitalClassicLEDControl,
+        channel: int,
+    ) -> None:
+        """Initialize an EHEIM Digital classicLEDcontrol light entity."""
+        super().__init__(coordinator, device)
+        self._channel = channel
+        self._attr_translation_placeholders = {"channel_id": str(channel)}
+        self._attr_unique_id = f"{self._device_address}_{channel}"
+        self._async_update_attrs()
+
+    @property
+    def available(self) -> bool:
+        """Return whether the entity is available."""
+        return super().available and self._attr_available
```

Comment:
Why? Just add what `_attr_available` is set to instead

Reply:
I dont't quite understand what you mean.

Reply:
`_attr_available` is set to a value, but why don't we evaluate that value here instead of setting it somewhere else? (I don't have a goal in mind, it just looks weird so I just wanted to mention it)

Reply:
Yeah, you are right, I did it that way right after I learned that `CoordinatorEntity` entities ignore `_attr_available`